demo-template/template/template.yaml

```diff
@@ -0,0 +1,137 @@
+# yaml-language-server: $schema=https://json.schemastore.org/yamllint.json
+
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: demo-template
+  title: Demo Template
+  description: Software Template used to demo Scaffolder features
+spec:
+  owner: backstage/maintainers
+  type: website
+
+  parameters:
+    - title: Fill in some steps
```

Comment:
Might be worth adding a nice markdown description here explaining what this template is going to do.

It's gonna be a nice description at the top, and you can probably link out to example site what it's gonna create if anyone wants to do it Run it

Reply:
I've added a better description but not sure if I went too detailed so can you give it another look please?